### PR TITLE
[#66] delete_layer in geojson

### PR DESCRIPTION
**Pull request by jsadler2**
seems like this fixes the error in writing the geojson

### CODE REVIEW

**Comment by jsadler2** [comment]
@lekoenig - can you give this a try?

**Comment by lekoenig** [comment]
Yep, thanks for looking into this! Glancing at #66 again, I must have been trying to build on my branch `36-match-segs` because one of those errors is related to the variable name change I made in PR #67 that's not yet in main (i.e. `offset` --> `bird_dist_to_subseg_m`).

File: 3_visualize.R [hunk 1]
```diff
@@ -65,7 +65,7 @@ p3_targets_list <- list(
     p3_well_observed_site_data_json,
     {
       filename = "3_visualize/out/well_observed_trn_val_test.geojson"
-      st_write(p3_well_observed_site_data, filename, append=FALSE)
+      st_write(p3_well_observed_site_data, filename, append=FALSE, delete_layer=TRUE)
```
Comment (lekoenig) [inline 3_visualize.R:68]: The geojson file may be writing OK, but `delete_layer = TRUE` still leaves me with these errors:
```
GeoJSON' layer not supported by driver `
Deleting layer `well_observed_trn_val_test' failed
```

After checking out the [documentation](https://r-spatial.github.io/sf/reference/st_write.html) for `st_write`, I wonder if instead we need to specify `delete_dsn`. I'm curious if the change below works for you (I also made the driver explicit and added a quiet message to suppress verbose output when running `tar_make`:

```suggestion
      st_write(p3_well_observed_site_data, filename, append = FALSE, delete_dsn = TRUE, driver = "GeoJSON", quiet = TRUE)
```

**Review by lekoenig — COMMENTED** [review]
I'm still getting the geojson error with `delete_layer = TRUE` but let me know what you think about the suggestion I've added here.

File: 3_visualize.R (hunk 1 above)
Comment (jsadler2) [reply 3_visualize.R:68]: That works great! Thanks for looking into this, @lekoenig

**Review by lekoenig — APPROVED** [review]
Looks good! Thanks for checking on this.

**Pull request merged**